app/models/landable/page.rb

```diff
@@ -42,7 +42,8 @@ class Page < ActiveRecord::Base
     delegate :republish!, to: :published_revision
 
     scope :imported, -> { where("imported_at IS NOT NULL") }
-    scope :sitemappable, -> { where("COALESCE(meta_tags -> 'robots' NOT LIKE '%noindex%', TRUE)") 
+    scope :sitemappable, -> { where("COALESCE(meta_tags -> 'robots' NOT LIKE '%noindex%', TRUE)")
```

Comment:
Line is too long. [97/80]<br>Block body expression is on the same line as the block start.